views/faqs.mustache

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Hello!</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+    <!-- import the webpage's stylesheet -->
+    <link rel="stylesheet" href="/style.css">
+    
+    <!-- import the webpage's javascript file -->
+    <script src="/script.js" defer></script>
+  </head>  
+  <body>
+    <h1>Donors FAQs</h1>
```

Comment:
Is the h1 tag looking as in the site? I think we need a container div, with a border and a margin. Take a look at the header in the attached image. 
![screenshot 2018-11-08 at 7 22 45 am](https://user-images.githubusercontent.com/26427672/48172777-23e15600-e327-11e8-9c3a-d170237c5224.png)